web-registry/src/pages/Projects/hooks/useProjectsSellOrders.utils.tsx

```diff
@@ -26,14 +35,22 @@ export const getPurchaseInfo = (
     .reduce((total, quantity) => total + quantity, 0);
 
   const prices = ordersForThisProject
-    .map(order => microToDenom(order.askAmount))
+    .map(order => {
+      const amount = microToDenom(order.askAmount);
+      const denomPrice = order.askDenom === REGEN_DENOM ? regenPrice ?? 0 : 1;
+
+      return amount * denomPrice;
+    })
     .sort((a, b) => a - b);
-  const priceMin = formatNumber(prices?.[0], 2);
-  const priceMax = formatNumber(prices?.[prices.length - 1], 2);
+  const priceMin = prices?.[0];
+  const priceMax = prices?.[prices.length - 1];
+  const priceMinDisplayed = priceMin > 0 ? formatNumber(priceMin, 2) : '';
+  const priceMaxDisplayed = priceMax > 0 ? formatNumber(priceMax, 2) : '';
```

Comment:
Maybe we can improve the `formatNumber` method. If the second digit behind the decimal is a zero, it is preferable to show the zero, and not round it. In other words,
"$7.90" is better than "$7.9"

Reply:
Sure, will handle it.